src/normal.cc

```diff
@@ -1848,7 +1848,7 @@ void combine_selection(const Buffer& buffer, Selection& sel, const Selection& ot
     }
 }
 
-template<typename Func>
+template<bool to_reg, typename Func>
```

Comment:
I'd prefer a StringView for the info box title to be passed there rathen than a bool template parameter, my issue here is that its the fact that we are saving to/from register is mostly controlled by Func, not by to_reg, which only selects the mesage.

Passing a normal StringView parameter would be fine.